tests/test_wheel.py

```diff
@@ -339,6 +339,23 @@ def test_install_subdir(wheel_install_subdir):
         }
 
 
+def test_install_symlink(wheel_symlinks):
+    artifact = wheel.wheelfile.WheelFile(wheel_symlinks)
+    # Handling of the exclude_files and exclude_directories requires
+    # Meson 1.1.0, see https://github.com/mesonbuild/meson/pull/11432.
+    # Run the test anyway to ensure that meson-python can produce a
+    # wheel also for older versions of Meson.
+    if MESON_VERSION >= (1, 1, 99):
```

Comment:
You don't use `exlude_files` or `exclude_directories` in your test, why this version check?